deploy/sre-prometheus/ocm-agent/obo-monitoring/100-oidc-missing.PrometheusRule.yaml

```diff
@@ -19,7 +19,8 @@ spec:
           # Clusters tend to have their `hypershift_cluster_invalid_aws_creds` set to > 0 while the HCP didn't finish the installation, thus we check
           # that the HCP is not rolling out in our expression (= hypershift_cluster_waiting_initial_avaibility_duration_seconds does not exist)
           # hypershift_cluster_waiting_initial_avaibility_duration_seconds stops being emitted once the HCP is rolled out
-          expr: sum by (mc_name, _mc_id, sector, region, env, namespace, exported_namespace, source, _id) (last_over_time(hypershift_cluster_invalid_aws_creds[10m])) > 0 unless on (exported_namespace) hypershift_cluster_waiting_initial_avaibility_duration_seconds unless on (exported_namespace) last_over_time(hypershift_cluster_deleting_duration_seconds[10m])
+          # This will be fixed with https://issues.redhat.com/browse/OCPBUGS-63353 
+          expr: (max by (mc_name, _mc_id, sector, region, env, namespace, exported_namespace, source, _id) (hypershift_cluster_invalid_aws_creds) == 1) unless on (exported_namespace) (hypershift_cluster_waiting_initial_avaibility_duration_seconds or hypershift_cluster_deleting_duration_seconds)
```

Comment:
This `oidc-missing` (RHOBS) `PrometheusRule` is deployed on MC clusters and is considered by COO (ex OBO) Prometheus pods.
At rules revaluation time [metrics have not yet been "remote written" to Dynatrace](https://gitlab.cee.redhat.com/service/osd-fleet-manager/-/blob/main/config/resources/managed-cluster-monitoring-stack.yaml#L31); this means that the `_mc_id`, `mc_name`, `region`, `sector`, `env` and `source` labels are not yet defined.
Please remove those labels from the `max by` list.